kbase/narrative
# DATAUP-330 merge develop -> truss

# Description of PR purpose/changes

This merges all the changes on the develop branch to the truss branch.
It also takes a Prettier pass over what's in the truss branch.
There's an intermediate commit at the end where there's a few file repairs due to some merge issues.

Also, due to some lingering issues on the truss branch of things, frontend tests will fail. There are also a few tests without `except` statements, so those will now throw an error. These will get patched up in a proceeding PR.

# Jira Ticket / Issue #
e.g. https://kbase-jira.atlassian.net/browse/DATAUP-X

# Testing Instructions
* Details for how to test the PR:
* The bundle should build and deploy on narrative-refactor
* Should be no other visible changes
- X Tests pass locally and in GitHub Actions

# Dev Checklist:

- X New and existing unit tests pass locally with my changes (not yet in this merge)
